patroni/dcs/consul.py

```diff
@@ -229,7 +229,8 @@ def __init__(self, config):
         self._register_service = config.get('register_service', False)
         self._previous_loop_register_service = self._register_service
         if self._register_service:
-            self._service_tags = config.get('service_tags', [])
+            self._service_tags = sorted(config.get('service_tags', []))
+            self._previous_loop_service_tags = self._service_tags
```

Comment:
If upon start the `register_service` isn't set in the config, the next call of `reload_config()` will fail due to undefined `self._service_tags`.